ibis/backends/__init__.py

```diff
@@ -1458,5 +1459,26 @@ class NoUrl:
 
     name: str
 
-    def _from_url(self, url: str, **_) -> ir.Table:
-        raise NotImplementedError(self.name)
+    def _from_url(self, url: str, **kwargs) -> BaseBackend:
+        """Connect to the backend with empty url.
+
+        Parameters
+        ----------
+        url : str
+            The URL with which to connect to the backend. This parameter is not used
+            in this method but is kept for consistency.
+        kwargs
+            Additional keyword arguments.
+
+        Returns
+        -------
+        BaseBackend
+            A backend instance
+
+        """
+        # Check if additional components are provided in the URL
+        if len(url.split("://")) > 1:
+            warnings.warn(
+                f"Additional components in the URL are ignored. url = '{url}'"
+            )
```

Comment:
I would remove the warning entirely, but if you'd like to keep it, I think you'll want to either use `urlparse` or check that the last element of the `split` call is not an empty string.

Reply:
Thanks for your feedback. I will remove the warning.